### PR TITLE
Fix broken links in Docs #3764

### LINKED ISSUE #3764: [DOCS] Change broken links in Docs
#### Description
The link in the related reading section of the [Service Mesh Pattern Management](https://docs.meshery.io/functionality/pattern-management) docs is giving a 404 error due to a wrong link
and the 'using helm' link in [Quick start with Kubernetes](https://docs.meshery.io/installation/platforms/kubernetes#using-kubernetes-manifests) doesn't redirect to the 'using helm charts' part of the page.

#### Expected Behavior
The links need to be changed.
Consider using this for the Service Mesh Pattern Management link : https://docs.meshery.io/reference/mesheryctl/pattern

#### Environment

- Host OS: Mac 
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker or Kubernetes

---

**Contributor Resources:**
- [Contributing](https://docs.meshery.io/project/contributing)

### PR DESCRIPTION
**Pull request by sgibson19**
**Description**

This PR fixes #3764 

**Notes for Reviewers**
* Please let know if there are any issues

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d94c2c0d83dc308fdf63fb2e655962b4926e6cb6 at: https://610f3a5314cf62c020b6b2cc--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 59c03800c3274aa39dddc7817cb412355536e5b3 at: https://610f3d07fd210ea8f164e110--meshery-docs-preview.netlify.app

File: docs/pages/functionality/patterns.md [hunk 1]
```diff
@@ -138,4 +138,4 @@ Deployed. Endpoint(s) available at: http://localhost:8000/catalog
 
 ## Related Reading
 
-- [`mesheryctl pattern`](/reference/mesheryctl/#service-mesh-pattern-configuration-and-management)
+- [`mesheryctl pattern`](https://docs.meshery.io/reference/mesheryctl/pattern)
```
Comment (leecalcote) [inline docs/pages/functionality/patterns.md:141]:
```suggestion
- [`mesheryctl pattern`]({{ site.baseurl }}/reference/mesheryctl/pattern)
```

**Comment by l5io** [comment]
🚀 Preview for commit 3894d2441d7002157785f7dcfb05534c36b6d38b at: https://6111a4397ea0e945d937b2d0--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f946e1edf0919bee832e58d1edbf558c69eafa83 at: https://6111b4d3de851d548e9b2a59--meshery-docs-preview.netlify.app

File: docs/pages/functionality/patterns.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/functionality/patterns.md:141]: Let's be sure to use the Jekyll / liquid tag in order to avoid hardcoding the site's URL.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Good work so far, @sgibson19

**Comment by l5io** [comment]
🚀 Preview for commit 281f1395af9d670c1bf2c8447320c2f0d31faca2 at: https://6111bfe4148ef56e09419396--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 024ce4352288388c41fb0fc8006bbb87a62db8f4 at: https://6111c0da21764b69ea4fc9f2--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]
Looks real good 👍

**Review by alphaX86 — APPROVED** [review]
LGTM 🎉 great work @sgibson19 👍

**Review by adithyaakrishna — APPROVED** [review]
LGTM 🎉 
Nice Work 🎈

**Pull request merged**